lib/cf/CloudControllerClient.js

```diff
@@ -275,6 +278,27 @@ class CloudControllerClient extends HttpClient {
     return this.getResource(`/v2/organizations/${org_guid}`, options);
   }
 
+  getMultiTenantServiceInstanceIdWithName(instance_name, org_guid, space_guid, service_name, options) {
```

Comment:
can you rename this method to `getServiceInstanceByName` - no need to have the `mutliTenant` added to the method signature as the same can be reused elsewhere in the code where one wants to retrieve instance details by name.